src/cosmicqc/analyze.py

```diff
@@ -24,7 +24,7 @@ def identify_outliers(
     feature_thresholds_file: Optional[str] = DEFAULT_QC_THRESHOLD_FILE,
     include_threshold_scores: bool = False,
     export_path: Optional[str] = None,
-) -> Union[pd.Series, pd.DataFrame]:
+) -> Union[pd.Series, SCDataFrame]:
```

Comment:
Thanks great questions: a Pandas DataFrame can still be used as input for this function but it will try to output an SCDataFrame to help with visualizations through reports or the image-based display in Jupyter. If a Pandas DataFrame was ever desired, it should be possible to convert back and forth via something like `pd.DataFrame(SCDataFrame())`. That said, I think this should be included in the nutshell notebook to help clarify things.